### DISCUSSION TITLE
Was Newton misled by Kepler?

### DISCUSSION THREAD

**Discussion by Pierre314159** [a-cosmology-group/acg#214] [Gravitation Models]
Not exactly cosmology-related, but a necessary follow-on of a year-old discussion involving mainly @sahil5d about orbital precession of periapsis, where I mentioned having some work in progress. Well, here it is:

http://dx.doi.org/10.2139/ssrn.4801844

Essentially, it is an empirical derivation of a law of gravitation that takes into account that orbits precess, the same way as Newton made his « inverse square law » by assuming that orbits were perfect ellipses, as per Kepler’s first law. It uses no hypotheses, and no adjustable parameter. It is amazing to see how close this gets to General Relativity’s prediction of precession, which needs a « curved spacetime » as a prior hypothesis.

Bottom line: if Kepler had done his job correctly and established his first law of planetary motions as precessing instead of closed ellipses, Newton’s law of gravitation would have predicted Mercury’s precession, LeVerrier would have discovered nothing abnormal in Mercury’s orbit, Einstein couldn’t claim his prediction of precession a « successful test of General Relativity », and precession wouldn’t be « relativistic » or « Schwarzschild » as it is called nowadays…

I think this puts a historical conundrum into a different perspective.

Thanks to @RedshiftDrift for his patience in numerous rounds of review !

**Comment by HanDeBruijn** [comment]
These are the [_three basic tests_](http://www.alternatievewiskunde.nl/cosmology/newton.htm) General Relativity has started with.

- Mercury's Perihelion Advance
- Gravitational deflection of light
- Gravitational redshift

Claim by the OP\:
>empirical observations, basic algebra, and parameters obtained from natural values match the predictions from General Relativity with great precision.

The universal law of gravitation as modified by @Pierre314159 obviously shall be applicable to the first test, because it has been designed in this way.

$$
F = GMm\left(\frac{1}{r^2}+\frac{6GM}{c^2r^3}\right)
$$

However, Pierre, does your MOKD theory also correctly predict the outcomes of the two other tests, [_Bending of Light_](http://www.alternatievewiskunde.nl/cosmology/bending.htm) in particular?

**Reply by Pierre314159** [reply]
Hi, Han!

Yes, three (3) tests of GR:

1) Perihelion advance of Mercury. As I have shown, any discrepancy between MOKD and GR that can be measured will necessitate observing a planet in orbit around a neutron star or the like, and that has never been done is not going to happen anytime soon. Strike one, two to go. Oh, and by the way, MOKD is not a theory: it is a law, the same way as Kepler's laws are laws. Also, note that my claim is "with great precision", and not "correctly predicts".

2) Deflection of light. I voluntarily refrained to go down that path to avoid the arguing about the controversies that surround this "verification" of GR, i.e.: how Einstein "postdicted" the deflexion angle by doubling his original calculation; how refraction in the solar atmosphere was overlooked; how Soldner got the right prediction as far back as 1801; how the 1919 eclipse expedition was orchestrated by Eddington in his efforts to promote Einstein's fame, etc. So strike two if you ask me, one to go. (MOKD's prediction is 0,9089 seconds of arc. I'll let you do the math)

3) Gravitational redshift. Could have been a thing as far back as 1590 if Galileo had thought it through when he showed that an object's mass is irrelevant to the speed at which it falls (provided it is relatively negligible). "Irrelevant" includes whether the object has a mass or doesn't, therefore, the question whether light (waves of particles) has a mass is moot. (Indeed, when calculating the deflection of light using Newtonian gravitation, the mass that photons are assumed to have eventually vanishes from the equations.) Anyways, from the moment it is accepted that light bends close to a mass, and it is observed that it does so without being diffracted into a spectrum, then it must be admitted that light slows down in a gravitational field. MOKD's prediction is $\sqrt{0,8\dot{3}-\frac{2GM}{rc^2}}$ (I'll let you do the math), slightly smaller then GR, but close ("with great precision"). For a GPS satellite, that's a difference of one tenth of one microsecond per second. 

Anyways, GR's verification by gravitational redshift had to wait for Pound and Rebka in 1959. What left had there been of a theory that had no experimental verification in 50 years? Strike three. Gr out. Batter up.

**Reply by HanDeBruijn** [reply]
Hi Pierre,

Ad 1.
>MOKD is not a theory: it is a law [ .. ]

In my (not so) humble opinion the formula below represents a theory, not so much a Keplerian law.

$$
F = GMm\left(\frac{1}{r^2}+\frac{6GM}{c^2r^3}\right)
$$

Ad 2.
>Soldner got the right prediction [ .. ] MOKD's prediction is 0,9089 seconds of arc.

According to [Doubling the Deflection](https://www.mathpages.com/rr/s8-09/8-09.htm) Soldner predicted _0.84 seconds of arc for a ray grazing the sun_.
(To be honest, I understand little of Einstein's math as presented at those _mathpages_.)

Ad 3.
>MOKD's prediction is $\sqrt{0,8\dot{3}-\frac{2GM}{rc^2}}$.

As discussed [elsewhere](https://github.com/orgs/a-cosmology-group/discussions/215#discussioncomment-9369554) the common prediction is $\sqrt{1-2GM/(r c^2)}$ , which is not too deviant from yours.

**Comment by HanDeBruijn** [comment]
What follows is from the section [Self Energy Field](http://www.alternatievewiskunde.nl/cosmology/nasa.htm) on my website.
We seek an analogue of the well known expression for the energy density $u_e$ in an electric field giving an $u_g$ for gravitation.

$$
u_e = \frac{1}{2}\epsilon_0 E^2 \quad \mbox{with} \quad E = \frac{Q}{4\pi\epsilon_0 r^2}
$$

where $\epsilon_0=$ electric constant, $E=$ electrical field strength, $Q=$ point charge, $r=$ distance.

$$
u_g = - \frac{1}{2}\frac{1}{4\pi G}\left(\frac{GM}{r^2}\right)^2 = - \frac{GM^2}{8\pi r^4}
$$

where $G=$ gravitational constant, $M=$ mass of spherical gravitating body, with radius $r_0$, $r=$ distance.
The minus sign is important.
Integrating the gravitational energy density $u_g$ over a sphere with radius $r \gt r_0$ gives:

$$
U_g = - GM^2 \int_{r_0}^r \frac{4\pi s^2 ds}{8\pi s^4} = - \frac{GM^2}{2}\left(\frac{1}{r_0}-\frac{1}{r}\right)
$$

Special Relativity ($U=Mc^2$) is giving rise to an corrected gravitating mass equal to

$$
M-\frac{GM^2}{2c^2}\left(\frac{1}{r_0}-\frac{1}{r}\right)
$$

Herewith Newton's law of universal gravitation is modified as follows.

$$
\frac{F}{m} = \frac{GM'}{r^2} + \frac{G^2M^2}{2c^2r^3}
$$

Given a radius $r_0$ and mass $M$ of e.g. the sun we have

$$
M' = M\left(1-\frac{GM}{2c^2r_0}\right) \approx M
$$

And so

$$
F \approx GMm\left(\frac{1}{r^2}+\frac{GM}{2c^2r^3}\right) \ne GMm\left(\frac{1}{r^2}+\frac{6GM}{c^2r^3}\right)
$$

With Newton's theory and Einstein's SR the **form is the same** as with @Pierre314159 's formula, but we are a factor of $12$ off.

**Comment by budrap00** [comment]
Hi Pierre,

I have to admit that I was a little put off by the title that seemed to accuse Kepler of scientific malfeasance - but then I realized it was just click bait since your conclusion exonerates him:

>Had he had access to measurements of planetary orbits with a precision in the order of LeVerrier’s, Johannes Kepler could have established the trajectory of orbits as elliptical hypotrochoids instead of perfect ellipses as the statement of his first law of planetary orbits.

The same can be said for Newton. Now you have taken the precession observations they didn't have and done what they would have done, recalculate the orbital geometry correctly. Good work Pierre! That's the way science is supposed to be done - the observations determine the model.

Now the question is, does this hypotrochoid geometry have any implications for the rotational curves of galaxies?

**Reply by HanDeBruijn** [reply]
>Now the question is, does this hypotrochoid geometry have any implications for the rotational curves of galaxies?

According to an [answer at MSE](https://math.stackexchange.com/questions/34308/confusion-with-the-various-forms-of-the-equation-of-second-degree/2115114#2115114) the _excentricity_ $\epsilon$ of a circle is zero: $\epsilon = 0$.
In the paper by Pierre equation (1) says that hypotrochoids in polar coordinates $(r,\theta)$ are given by

$$
r = \frac{1}{1-\epsilon\cos(\omega\theta)} \quad \Longrightarrow \quad r = 1 \quad \mbox{for circles}
$$

Thus any hypotrochoid corresponding to a circle in the Keplerian picture is simply a circle.
Now it's a fact that [simple models for galaxies](http://www.alternatievewiskunde.nl/melkweg/index.htm) use a bunch of circles for the mathematics, not $\epsilon\ne 0$ ellipses.
This means that there is little chance that the theory by Pierre will lead to new insights _for the rotational curves of galaxies_.

**Reply by Pierre314159** [reply]
Hi, Bud!

> your conclusion exonerates him

Of course, poor Johannes only has Tycho's data and instruments capable at the most of a [precision of _tens_ seconds of arc](https://www2.hao.ucar.edu/education/scientists/tycho-brahe-1546-1601), far from the _tenths_ of second of arc necessary to view Mercury's precession. Then again, it may well be that he saw the precessions, but chose to make obrits ellipses anyway because he wasn't totally free from the Ptolemeian philosophy that required everything heavenly to be perfect. Anyway, I "exonerated" him as you say, because in the end, that is unimportant. What's important is that orbits are not ellipses.

> Good work Pierre!

Thank you !

> Now the question is, does this hypotrochoid geometry have any implications for the rotational curves of galaxies?

I doubt it because what the math says is that gravity is stronger than the inverse square of distance at short distances, and that is the opposite of what would be required to explain galaxy rotation. Anyways, I find the problem of galaxy rotations uninteresting because it is poorly framed.

**Reply by HanDeBruijn** [reply]
@Pierre314159 . @budrap00 .
>Anyways, I find the problem of galaxy rotations uninteresting because it is poorly framed.

_Uninteresting_ ?? _Poorly framed_ ?? Pierre, I really don't understand what you are trying to say here. Could you please formulate this again in somewhat more elaborate prose, for a native Dutch speaker like me.

Can't resist. Velocities $v$ in a galaxy: $v_P$ according to Pierre compared with $v_N$ according to Newton. Circles with radius $r$ assumed.

$$
m.a = m\frac{v_P^2}{r} = GMm\left[\frac{1}{r^2}+\frac{6GM}{c^2r^3}\right] \quad ; \quad v_N = \sqrt{\frac{GM}{r}}
$$

$$
v_P = \sqrt{\frac{GM}{r}}\sqrt{1+\frac{6GM}{c^2r}} \approx \sqrt{\frac{GM}{r}}\left[1+\frac{1}{2}\frac{6GM}{c^2r}\right] = v_N\left[1+3\left(\frac{v_N}{c}\right)^2\right]
$$

Take the speed of the sun in the milky way as a typical $v_N \approx 300\mbox{ }km/s$ and $c \approx 300,000 \mbox{ }km/s$ then the relative error is

$$
\frac{v_P-v_N}{v_N} \approx 3 \times \left(\frac{300}{300,000}\right)^2 = 3\times 10^{-6}
$$

Consequently, indeed, Pierre's velocities can never explain the flat velocity profiles in a galaxy:

![Rotation_curve_of_spiral_galaxy_Messier_33_(Triangulum)](https://github.com/a-cosmology-group/acg/assets/135433222/9883569c-8818-495e-8c4f-63c04cbec9a7)

**Comment by sahil5d** [comment]
Hi Pierre, Maneff's force equation
https://arxiv.org/pdf/1611.09660
https://arrow.tudublin.ie/cgi/viewcontent.cgi?article=1094&context=scschmatart
may be relevant to the historical conundrum.

Your work as well as Maneff's are super important as they show what's really going on to explain Mercury's precession: multiplying `GM/r^2` by `(1 + k*GM/(rc^2))`, and there is a k because it fits the data. So few physicists articulate this.

GR says it has a justification for 3, https://arxiv.org/pdf/1106.1568, but it's suspect because it hides an r^2 in the angular momentum term, l^2, which is (mvr)^2, and then compensates with a 1/r^3, and so when taking the derivative conveniently gets the coefficient 3.

Returning to our year-old discussion, I still think a `k * v^2/c^2` modification might be more accurate than a `k * GM/(rc^2)` modification, because
- a `v^2/c^2` term is in Sommerfeld's quite successful elliptical, precessing model of electron orbits
- the `(1 + k*GM/(rc^2))` factor hasn't yet been detected in the lab

(*Edited post. GR doesn't say k=6 but k=3.)

**Reply by HanDeBruijn** [reply]
Interesting post. The name of Maneff/Manev is also mentioned in the book [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf) by Amitabha Ghosh. Referencing is followed by two equations

$$
m=m_0\exp\left[\frac{GM}{c^2r}\right] \qquad (2.23)
$$

$$
F=\frac{GMm_0}{r^2}\left[1+\frac{3GM}{c^2r}\right] \qquad (2.24)
$$

The equation in the paper [Manev Potential and General Relativity](https://arrow.tudublin.ie/cgi/viewcontent.cgi?article=1094&context=scschmatart) corresponding with (2.23) is $m=m_0\exp(GM/r)$ ; it is not dimensionally correct. Equation (2.24) and equation (1) in the paper about Maneff are identical. However, I cannot see how (2.24) can be derived from (2.23).
> GR says it has a justification for the 6 [ .. ] but it's suspect because [ .. ] when taking the derivative conveniently gets the coefficient 3.

Yes, and NASA says there is a justification for the $1/2$ in _[A Possible Scalar Term Describing Energy Density in the Gravitational Field](https://www.grc.nasa.gov/www/k-12/Numbers/Math/Mathematical_Thinking/possible_scalar_terms.htm)_. There are at least three alike theories, differing by a factor ranging from 1/2 via 3 to 6.

Sahil, it's easy to improve your formatting by adding a few dollar signs:
```
I still think a $k \times v^2/c^2$ modification might be more accurate than a $k \times GM/(rc^2)$ modification
```
Giving
>I still think a $k \times v^2/c^2$ modification might be more accurate than a $k \times GM/(rc^2)$ modification

Does this correspond with equation (55) in [When effective theories predict](https://arxiv.org/pdf/1106.1568) ? There is an alike approach in [Modified Newton’s Gravitational Theory to Explain Mercury Precession and Light Deflection](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/Keli_Kou.pdf): see equation (2.8) in that paper.
>the $(1 + 6GM/(rc^2))$ factor hasn't yet been detected in the lab

How so? Isn't the solar system a giant lab of nature?

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn 2.24 is a Taylor Series expansion of 2.23 but for that $x$ in $e^x$ must be dimensionless else you can't expand it. Yes, there seems to be a problem with dimensions and/or units.

I would be more interested how they got to 2.23 in the first place. I shall have a look.
In my own model of absorption
$I = I_0*e^{-μρx}$ with $μ = 4πG/c^2$ as an eventual outcome.
but it's a long way of derivations from Flux to Force, and it's tough to put it back into the exponential and try make sense of it. 

Point is, I think Newton is just a result of the first term of this expansion, and Ghosh includes a second term.
Looking at 2.23 again, and including my $G = μc^2/4π$, then 2.23 looks like
$m = m_0*e^{μM/4πr}$
still nasty ....

**Reply by sahil5d** [reply]
> Does this correspond with equation (55) in 'When effective theories predict' (arxiv.org/pdf/1106.1568) ? There is an alike approach in 'Modified Newton’s Gravitational Theory to Explain Mercury Precession and Light Deflection' (alternatievewiskunde.nl/cosmology/ACG_refs/Keli_Kou.pdf)

They have the `k * v^2/c^2` form, so they are similar. But specifically, Gerber's idea has an issue of explaining the modification as due to  "gravity field changes happen at c", which I think is unnecessary and wrong (https://github.com/orgs/a-cosmology-group/discussions/82#discussioncomment-7129869). And Kou's paper does the same suspect derivation as GR, by hiding an r^2 in the angular momentum term, l^2.

I only vouch for the `k * v^2/c^2` form. I don't yet know why it works, and I don't yet know why a certain k is the best fit.


> Isn't the solar system a giant lab of nature?

It is, but observations of Mercury don't distinguish `k * GM/(rc^2)` from `k * v^2/c^2`. But an on-the-ground lab could have detected a `k * GM/(rc^2)` modification and hasn't so far.

**Reply by sahil5d** [reply]
> it's easy to improve your formatting by adding a few dollar signs:

Thanks, but I prefer code-like monospace text to LaTeX formatting. It's also easier to read in plaintext and email.

**Reply by HanDeBruijn** [reply]
>observations of Mercury don't distinguish $6GM/(rc^2)$ from $6v^2/c^2$.

We've seen that issue before [in thread 215](https://github.com/orgs/a-cosmology-group/discussions/215#discussioncomment-9369554):
>gravity potential is linked to escape velocity $v_e$ via kinetic energy:

$$
\frac{GMm}{R_e} = \frac{1}{2}mv_e^2 \quad \Longrightarrow \quad v_e^2 = \frac{2GM}{R_e}
$$

This would mean that we _don't distinguish_ ${\bf 6}GM/(rc^2)$ _from_ ${\bf 3}v^2/c^2$ .

**Reply by Francois-Zinserling** [reply]
That's half of the measured result ...
the same error when Newton is used for predicting the bending of light?

**Reply by HanDeBruijn** [reply]
>the same error when Newton is used for predicting the bending of light?

Or calculate with circular motion:

$$
\frac{GMm}{r^2} = m\frac{v^2}{r} \quad \Longrightarrow \quad 6\left(\frac{v}{c}\right)^2 = \frac{6GM}{rc^2}
$$

**Reply by HanDeBruijn** [reply]
@Francois-Zinserling .
The formulas are remarkably close to somehing else.

$$
m=m_0\exp\left[\frac{GM}{c^2r}\right] \qquad (2.23)
$$

According to (our version of) The Variable Mass Theory we have:

$$
\frac{m}{m_0} = e^{H/c.r}
$$

For galaxies there is something peculiar with the rotation curves:

![verlinde](https://github.com/a-cosmology-group/acg/assets/135433222/370cabe5-a350-444b-84c3-46bfc584dddf)

And so (with $R\to r$ ) there might be a relationship with (y)our theory (?) \:

$$
\frac{GM}{r^2} = cH \quad \Longleftrightarrow \quad \frac{GM}{c^2r} = H/c.r
$$

**Reply by Pierre314159** [reply]
@sahil5d 

> It is, but observations of Mercury don't distinguish 6GM/(rc^2) from 6v^2/c^2. But an on-the-ground lab could have detected a 6GM/(rc^2) modification and hasn't so far.

They are equivalent when $v$ is the orbital velocity. If $v$ is some scalar velocity, such as in Kou's paper, then I'd say that there is nothing physical to justify that.

$\frac{6G}{c^2}$ being in the order of $4,4E^{-27}$, you would need a precision of that order times the mass/distance involved in the experiment, so good luck with that!  A much more crucial question is: why does k=6 fits oservations when GR says that k=3?

**Reply by sahil5d** [reply]
Hi Pierre, sorry I mistook GR saying 6 when it says 3.

Still I stand by my point that the GR derivation is suspect. If one looks at eq 54 or eq 68 in https://arxiv.org/pdf/1106.1568, and takes the derivative,
`d/dr (-GMm/r  (ℓ^2/(m^2 c^2) / r^2))`
` = d/dr (-GMm/r^3  ℓ^2/(m^2 c^2))`
` = 3 GMm/r^4  (m^2 v^2 r^2)/(m^2 c^2)`
` = 3 GMm/r^2  v^2/c^2`
An r^2 is hidden in the angular momentum term, ℓ^2, and with the 1/r^3 compensation, taking the derivative, conveniently gets the 3.

> why does k=6 fit

Could you confirm where your data is from? The data in https://arxiv.org/pdf/1106.1568, Table 2, supports k=3.

> you would need a precision of that order times the mass/distance involved in the experiment, so good luck with that! 

Ah, yes you are correct.

> nothing physical to justify that

There is something physical to justify a preference for a `k * v^2/c^2` term over a `k * GM/(rc^2)` term which is that a `v^2/c^2` term appears in Sommerfeld's elliptical, precessing model of electron orbits, which was an improvement on Bohr's model. Also, it's not like we haven't seen a gamma term in other areas of physics.

**Reply by Pierre314159** [reply]
> why does k=6 fits oservations when GR says that k=3?

I take that back: Schwarzschild's force-equivalent does say k=6 (with some refinement!) according to [this ](https://biblioteca.org.ar/libros/90154.pdf)(equation 15, when $V^2$ is replaced with the Newtonian orbital velocity $\frac{GM}{r}$). [Burchell's](http://www.alternativephysics.org/book/SchwarzschildForce.htm) derivation must be incorrect, although I can't find where the error is...
